### ISSUE TITLE
Few feature request: notes, speed dial, sub collection, auto tag & folder suggest, side pane bookmarks

### ISSUE THREAD

**Issue by Alexnoj** [linkwarden/linkwarden#426]
- option add note - describe is for describe the website not for leave notes.
- speed dial dashboard with folders and sub folders (like in Vivaldi browser). bookmark page to speed dial by pin (for not creating double)
- in speed dial option set thumbnail and set big icon/image.
- option to create sub collections
- auto suggest tags and folders like in raindrop.
- browser extension side pane bookmarks
- to open up option for community create themes.

Thanks

**Comment by e-p-s** [comment]
Yes, so we can replace Homer/Dashy/Homepage with Dynamic content. 

+1 from me

**Comment by jcolinger3** [comment]
Subcollection or subfolders +1
